lib/Bio/EnsEMBL/DataCheck/Checks/CheckMethodLinkSpeciesSetNames.pm

```diff
@@ -64,6 +65,7 @@ sub tests {
         is( $species_set_name, $mlss_p1, $desc_2 );
       }
     }
+    next if $species_set->first_release < 80;
```

Comment:
```suggestion
    next if $species_set->first_release <= 80;
```

Reply:
In plants there was an issue in that many species_sets below first_release 81 (80 is included) have no name at all.